tc/examples/tensordot.cc

```diff
@@ -21,18 +21,33 @@
 #include <glog/logging.h>
 #include <gtest/gtest.h>
 
-#include <ATen/ATen.h>
-
-#include "tc/aten/aten_compiler.h"
-#include "tc/autotuner/genetic_autotuner_aten.h"
+#include "tc/aten/aten.h"
+#include "tc/aten/aten_autotuner.h"
+#include "tc/aten/aten_compiler_new_api.h"
+#include "tc/autotuner/genetic_search.h"
+#include "tc/core/cpu/cpu_mapping_options.h"
+#include "tc/core/cpu/cpu_tc_executor_new_api.h"
 #include "tc/core/cuda/cuda_mapping_options.h"
+#include "tc/core/cuda/cuda_tc_executor_new_api.h"
 #include "tc/core/flags.h"
 
-#include "../test/test_harness_aten_cuda.h"
-
 DEFINE_string(proto_path, "", "Filename to load and store proto cache ");
 
-TEST(TensorDot, SimpleAutotune) {
+template <typename Backend>
+at::Tensor makeATenTensor(at::ArrayRef<long int> sizes);
```

Comment:
it's a shame this thing is hidden in a particular example, could be useful elsewhere

Reply:
it's pretty vanilla .. we could add a few more convenience functions to aten.h, not here